### PR TITLE
Reformat + switch from `private` to `#`

### PR DESCRIPTION
**Pull request by sadym-chromium**
DRAFT PR.

1. Reformat all the source tree to align with prettier fromatter.
2. Switch the rest of `private _someField` to `#someField`.

### CODE REVIEW

File: src/cdp/websocketTransport.ts [hunk 1]
```diff
@@ -19,26 +19,28 @@ import { ITransport } from '../utils/transport';
...
-  constructor(private _ws: WebSocket) {
-    this._ws.on('message', (message: string) => {
-      if (this._onMessage) {
-        this._onMessage.call(null, message);
+  constructor(webSocket: WebSocket) {
+    this.#webSocket = webSocket;
+    this.#webSocket.on('message', (message: string) => {
+      if (this.#onMessage) {
+        this.#onMessage.call(null, message);
       }
```
Comment (thiagowfx) [inline src/cdp/websocketTransport.ts:30]: This could be:

```
this.#onMessage?.call(null, message);
```

File: src/bidiMapper/mapper.ts [hunk 2]
```diff
@@ -109,26 +109,26 @@ function _createCdpConnection() {
...
-    private _onMessage: ((message: string) => void) | null = null;
+    #onMessage: ((message: string) => void) | null = null;
 
     constructor() {
       window.onBidiMessage = (message: string) => {
-        if (this._onMessage) {
-          this._onMessage.call(null, message);
+        if (this.#onMessage) {
+          this.#onMessage.call(null, message);
         }
```
Comment (thiagowfx) [inline src/bidiMapper/mapper.ts:118]: ditto

File: src/bidiMapper/mapper.ts [hunk 3]
```diff
@@ -73,33 +73,33 @@ const _waitSelfTargetIdPromise = _waitSelfTargetId();
...
   class WindowCdpTransport implements ITransport {
-    private _onMessage: ((message: string) => void) | null = null;
+    #onMessage: ((message: string) => void) | null = null;
 
     constructor() {
       window.cdp.onmessage = (message: string) => {
-        if (this._onMessage) {
-          this._onMessage.call(null, message);
+        if (this.#onMessage) {
+          this.#onMessage.call(null, message);
```
Comment (thiagowfx) [inline src/bidiMapper/mapper.ts:88]: ditto

**Review by thiagowfx — APPROVED** [review]

**Comment by thiagowfx** [comment]
Could you also update the commit message to include the rationale / purpose / motivation behind the refactoring?

**Comment by sadym-chromium** [comment]
abandoning as it does not provide too much value

**Pull request closed without merging**